x-pack/plugins/actions/common/routes/connector/response/schemas/v1.ts

```diff
@@ -98,3 +98,55 @@ export const connectorTypesResponseSchema = schema.object({
     meta: { description: 'Indicates whether the action is a system action.' },
   }),
 });
+
+export const connectorExecuteResponseSchema = schema.object({
+  connector_id: schema.string({
+    meta: {
+      description: 'The identifier for the connector.',
+    },
+  }),
+  status: schema.oneOf([schema.literal('ok'), schema.literal('error')], {
+    meta: {
+      description: 'The outcome of the connector execution.',
+    },
+  }),
+  message: schema.maybe(
+    schema.string({
+      meta: {
+        description: 'The connector execution error message.',
+      },
+    })
+  ),
+  service_message: schema.maybe(
+    schema.string({
+      meta: {
+        description: 'An error message that contains additional details.',
+      },
+    })
+  ),
+  data: schema.maybe(
+    schema.any({
+      meta: {
+        description: 'The connector execution data.',
+      },
+    })
+  ),
+  retry: schema.maybe(
+    schema.nullable(
+      schema.oneOf([schema.boolean(), schema.string()], {
+        meta: {
+          description:
+            'When the status is error, determines whether the connector execution will retry .',
+        },
+      })
+    )
+  ),
+  errorSource: schema.maybe(
```

Comment:
seems like this doesn't conform the rest of our response schema which is snake case not camel case? I think that's what we return now but maybe we should be transforming it to `error_source`? Don't know if that is a breaking change thought 😬

Reply:
We spoke offline and decided not to change this field, as we believe it's a breaking change.